### ISSUE TITLE
For some posts the comments don't show up

### ISSUE THREAD

**Issue by nima1981** [fr4nzap/zapthreads#43]
See comments on this post: https://nostr.band/note1vugj5sc4y9aths2d2gla8y65xjn02wz0rcxeghs5r66mcngcagnqd0f82f

But they don't show up in the comments section here:
https://dissentwatch.com/boost/?boost_post_id=668592

I've included all relays showing up on nostr.band in the relay argument for the embed script
wss://relay.nostr.band/
wss://soloco.nl/
wss://xn--relao-dra1a.nostr.bg/
wss://realy.nostr.bg/
wss://really.nostr.bg/
wss://relar.nostr.bg/

**Comment by fr4nzap** [comment]
Can you try clicking the "..." three dots icon, cleaning the cache and try again?

**Comment by fr4nzap** [comment]
Actually, the article is on those relays, not the comments. Go to the first comment on nostr.band and you'll see it's on `wss://broadcaster.nostr.net/ wss://relay.mostr.pub./`

**Comment by fr4nzap** [comment]
I will work on the outbox model #29 and this should pick it up but only if the user has those relays listed per NIP-65

**Comment by nima1981** [comment]
> Can you try clicking the "..." three dots icon at the very bottom, cleaning the cache and try again?

Where are th

> Actually, the article is on those relays, not the comments. Go to the first comment on nostr.band and you'll see it's on `wss://broadcaster.nostr.net/ wss://relay.mostr.pub./`

relay.mostr.pub is included in the relays fed to the zapthreads plugin. I've also added broadcaster.nostr.net now. But the comments still aren't showing up. Where is this three dots link to clear he cache? I'm not seeing that.

**Comment by nima1981** [comment]
I've found the three dots and clicked the button to refresh the cache. Not sure if it did anything though.

**Comment by fr4nzap** [comment]
Is it still broken? Can you share a link so I can take a look?

**Comment by nima1981** [comment]
Yes, it's still broken: https://dissentwatch.com/boost/?boost_post_id=668592

I've now included the nocomments plugin above zapthreads for comparison

Also, zapthreads doesn't show up on iphone browsers.

**Comment by fr4nzap** [comment]
Sorry for the delay. Will be looking at this soon
